build/codegen/codegen.sh

```diff
@@ -16,7 +16,7 @@
 
 GROUP_VERSIONS="ceph.rook.io:v1"
 
-scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
I consider this orthogonal to the purpose of reverting back to Rook's header. Let's nix any non-necessary changes like this and the shebang at the top.